2.adoc

```diff
@@ -73,4 +73,151 @@ Optional description
 . Minus B
 
 
+=== Sublime Text
+
+* Multiplatformní desktopový editor textových souborů a zdrojových kódů s vysokou mírou přizpůsobení prostřednictvím pluginů vyvíjených komunitou
+* link:+https://www.sublimetext.com/+[https://www.sublimetext.com/]
+
+==== Porovnání rozdílů
+* Nejedná se o webovou komponentu, ale o desktopovou aplikaci
+* Univerzální editor, není omezený pouze na markdown jazyky
+* Defaultně neumí zobrazovat preview markdown dokumentu ani jeho outline (možnost doinstalovat pluginy)
+* Obsahuje propracovanou command pallete, přes kterou lze obsluhovat téměř všechna nastavení
+
+==== Zhodnocení UI
```

Comment:
Stejne jako u #20 mi pripada ze by bylo prehlednejsi vzhledem k hodne obrazkum, predelat root odrazky (Základní UI po instalaci, ...) na nadpisy a pododrazky na root odrazky.

Mozna je to spatne v Acme template, za coz se omlouvam. Ted jak je to videt s temi obrazky tak jsou vnorene senzamy hrozne zmatene.

Reply:
Problém je nejspíš v tom, že ty obrázky nejsou součástí položky seznamu.

```
* Lorem ipsum
+
image::path.png[...]
````

Reply:
To je sice pravda a mozna to tomu trochu pomuze, ale i tak jsou obrazky dost velke na to, ze napohled rozsypou strukturu seznamu. Proto bych i tak pouzil nadpisy.

Reply:
To nerozporuju. :) Já se na to v zatím díval jen z mobilu, na kód.